apps/marketplace/components/helpers.js

```diff
@@ -31,23 +31,24 @@ export const getResponsesFileSizeAndType = (bytes, lot) => {
   return result
 }
 
-export const nextWeekDay = date => {
+const getPreviousWeekDay = date => {
```

Comment:
Technically it would be `getPreviousWeekDayIfWeekend` as it only gets the previous weekday if the date itself is not a weekday, but that's splitting hairs.